packages/rlc-common/src/metadata/buildSampleEnvFile.ts

```diff
@@ -0,0 +1,23 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+import { RLCModel } from "../interfaces.js";
+
+const sampleEnvText = `
+# App registration secret for AAD authentication
+AZURE_CLIENT_SECRET=
```

Comment:
Could you check if this is still valid?

Reply:
@HarshaNalluru could you hep check if tthis is expected as of the security concern of spring groove